src/components/Editorial/Ctas.tsx

```diff
@@ -0,0 +1,44 @@
+import Button, { type ButtonProps } from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { type CommonProps } from 'types/components';
+
+interface CtaButton {
+  text: string;
+}
+
+export interface EditorialCtaProps extends CommonProps {
+  ctaButtons?: CtaButton[];
+}
+
+export const Ctas = ({ ctaButtons, theme }: EditorialCtaProps) => {
+  const buttonsTheme: ButtonProps[] = [
+    {
+      color: theme === 'dark' ? 'negative' : 'primary',
+      variant: 'contained',
+    },
+    {
+      color: theme === 'dark' ? 'negative' : 'primary',
+      variant: 'outlined',
+    },
+  ];
+
+  return ctaButtons?.length ? (
+    <Stack
+      direction={{ md: 'row-reverse', xs: 'row' }}
+      justifyContent="left"
+      spacing={2}
+    >
+      {ctaButtons.map((button: CtaButton, i) => {
+        return (
+          <Button
```

Comment:
Maybe it's always better to _abstract_ a component... if we have a FE that's use React Router, how can pass a `Link` Component?

Reply:
I did a similar thing on the hero component. button could be a type of CtaButton or JSX.Element

https://github.com/pagopa/pagopa-editorial-components/pull/4/files#diff-1cadf8397b7345b9f320a7c7e007a8bc697f2c9deec3085de9bd17e057ac0f0aR63-R70